exercises/practice/leap/test.pony

```diff
@@ -11,8 +11,13 @@ class iso _TestLeap is UnitTest
   fun name(): String => "leap/Leap"
 
   fun apply(h: TestHelper) =>
+    // Testing if 1996 is a leap year
     h.assert_true(Leap(1996), "Testing a leap year")
+    // Testing if 1997 is a leap year
     h.assert_false(Leap(1997), "Testing an odd year")
+    // Testing if 1998 is a leap year
     h.assert_false(Leap(1998), "Testing a non-leap even year")
+    // Testing if 1900 is a leap year
     h.assert_false(Leap(1900), "Testing a century")
+    // Testing if 2000 is a leap year
```

Comment:
These comments are so identical to the code statements that they are probably not more helpful than noise.  The tests are very clear on their own.

Reply:
It's intentional. 

The test runner pulls the line-number from the PonyTest harness' output and extracts the "name" of the test from the comment one line above it.

In some exercises it doesn't look neat.

Maybe an explanatory comment in these files would make sense.

Reply:
The blur of "Sometimes comments are code".  A comment in these files may make sense, perhaps for the first few tests at least.  After seeing it a few times as a learner, then it becomes familiar, and not as necessary.

Reply:
It's not there for the learner, it's there for the test-runner.

Reply:
So the tests for pony are not delivered to the student?  They are not there to be read by the learner?  To give information for the learner?

This would be different here than it is for other tracks.  I was not aware.

Reply:
Clearly we are talking past each other. I'll close this PR, submit just the change to the config.json and wait for the test-runner to be activated so I understand how it works.

Once I understand how that works I'll re-write the test-runner and re-architect the exercises.